lib/organisms/disqus-comments/README.md

```diff
@@ -0,0 +1,32 @@
+# Lean Patterns - Organism - Disqus Comments
+
+The component is part of the 'lnPatterns' module.
+
+It will render a Disqus Comments section where used.
+
+## Prerequisites
+
+In order to use this component you must include the following in your project:
+
+An NPM dependency on the [angular-utils-disqus library](https://github.com/michaelbromley/angularUtils/tree/master/src/directives/disqus):
```

Comment:
Because that would mean always loading the lib, even when you don't want to use it. In many cases we won't use this component.